sandbox

```diff
@@ -31,6 +31,18 @@ export INDEXER_BRANCH=""
 export INDEXER_SHA=""
 export INDEXER_DISABLED=""
 
+##################################################
+# Determine the correct docker command (or fail) #
+##################################################
+if [ -x "$(command -v docker-compose)" ]; then
+  DOCKER_COMPOSE="docker-compose"
+elif [ "$(docker compose help &>/dev/null; echo "$?")" -eq 16 ]; then
```

Comment:
Haha, I know, that's random. You'll have to ask the docker crew, though! For some reason they made `docker compose help` exit with code 16. (`docker non-existent-command help` exits with status 1, so this could also be written as `-ne 1`, but I tend to like explicit values, rather than negations.)